### PR TITLE
fix: restrict FSD2019Kaggle eval to curated subset only

### PR DESCRIPTION
**Pull request by AdnanElAssadi56**
The noisy subset has automatically-extracted labels with known label noise, it was intended as additional training data in the original FSDKaggle2019 Kaggle competition, not as a test set. Evaluating on noisy bounds lrap by the label-noise floor regardless of model quality, which confounds comparison across models.

If you add a model or a dataset, please add the corresponding checklist:

* [dataset checklist](https://embeddings-benchmark.github.io/mteb/contributing/adding_a_dataset/#submit-a-pull-request)
* [model checklist](https://embeddings-benchmark.github.io/mteb/contributing/adding_a_model/#submitting-your-model-as-a-pr)

### CODE REVIEW

**Comment by Samoed** [comment]
I think we can keep this, just to not add to "benchmark"

**Comment by AdnanElAssadi** [comment]
> I think we can keep this, just to not add to "benchmark"

how? leaderboard will aggeragate

**Comment by Samoed** [comment]
> how? leaderboard will aggeragate

We can select different subsets/splits for each tasks and on leaderboard will be only selected splits and subsets https://github.com/embeddings-benchmark/mteb/blob/b8ade6e386166463965e9f788278f2db311618e0/mteb/benchmarks/benchmarks/benchmarks.py#L70-L78

**Comment by AdnanElAssadi** [comment]
> > how? leaderboard will aggeragate
> 
> We can select different subsets/splits for each tasks and on leaderboard will be only selected splits and subsets
> 
> https://github.com/embeddings-benchmark/mteb/blob/b8ade6e386166463965e9f788278f2db311618e0/mteb/benchmarks/benchmarks/benchmarks.py#L70-L78

oh, got it
